lib/sprockets/sass/utils.rb

```diff
@@ -57,6 +57,12 @@ def get_class_by_version(class_name, version = version_of_sprockets)
           nil
         end
 
+        def get_less_class_by_version(class_name, version = version_of_sprockets)
+          constantize("Sprockets::Less::V#{version}::#{class_name}")
```

Comment:
The Less classes are defined in ```sprocket-less``` gem,  and this is needed for integration with that gem ( As i said in the description, this is the pull request for LESS with sprockets 3 https://github.com/lloeki/sprockets-less/pull/5 ) 

I know this is a bit ugly , but if that gem is not required, this won't affect anything. And even if you require it but you don't have any less files, those importers from that gem will never be called and neither this code.

Only if you use LESS inside SASS or SCSS files, this code will execute. 

I honestly found this when i wanted to integrate this gem in one of my applications, i realized i was using LESS inside SASS files ( by requiring them) and with sprockets 2.x that worked fine.

This add support for this type of usage for newer versions of sprockets too. 

Hope now is much more clear. Let me know if you have any other questions. I will be very glad to help any way i can :)